modules/hpn-lite/tests/unit/server-public-key-accessor-test.es

```diff
@@ -90,16 +90,12 @@ export default describeModule('hpn-lite/server-public-key-accessor',
         await storage.set(someStorageKey, entry);
       };
 
-      const oldCrypto = global.crypto;
+      const oldImportKey = global.crypto && global.crypto.subtle.importKey;
       beforeEach(async function () {
-        /* eslint-disable-next-line global-require */
-        global.crypto = global.crypto || {
-          subtle: {
-            importKey(...args) {
-              return MOCKS.importKey(...args);
-            },
-          }
-        };
+        if (!global.crypto) {
```

Comment:
As it is currently written, the tests have the side effects to change
`global.crypto === undefined` (before tests) to `global.crypto !== undefined` (after tests)